### DISCUSSION TITLE
Don't upgrade Perl on synology...

### DISCUSSION THREAD

**Discussion by tutle** [LMS-Community/slimserver#712] [General]
Hello, very disappointed since update of the  Perl Package (on my Synology 1513+)
LMS don't start anymore (I lost all my family configuration by trying uninstal, reinstall) And impossible to downgrade Perl on DSM 6.2.4
I solved by downgrading perl version (uninstall and manual install of 5.24.070)

**Comment by RobMeerwijk** [comment]
Hi, you can install docker on your Synology and run LMS from Docker. Works like a breeze. https://registry.hub.docker.com/r/lmscommunity/logitechmediaserver/ contains a section for Synology.
